apisix/stream/xrpc/runner.lua

```diff
@@ -38,6 +38,22 @@ local function open_session(conn_ctx)
 end
 
 
+local function put_req_ctx(session, ctx)
```

Comment:
This function more like releasing the req ctx

Reply:
I get the idea of these verbs from Go's sync.Pool: https://riptutorial.com/go/example/16314/sync-pool